pwndbg/commands/nearpc.py

```diff
@@ -97,6 +99,10 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
 
         line   = ' '.join((prefix, "%#x" % i.address, s or '', asm))
 
+        # Highlight the current line if the config is enabled
```

Comment:
I could switch it back but felt like being usefull when later we may get syntax highlighting for asm output, therefor it felt more appropriate to do it here. The example of the /x command pull request looks neat